contrib/completion/bash/docker

```diff
@@ -3668,6 +3668,10 @@ _docker_swarm_init() {
 			__docker_complete_local_interfaces
 			return
 			;;
+		--default-addr-pool)
+			COMPREPLY=( $( compgen -W "list of subnet addresses followed by subnet size" -- "$cur" ) )
+			return
+			;;
```

Comment:
This is not correct. The argument to `-W` is the list of possible completions.
Furthermore, I cannot think of a useful completion, so it's best not to complete anything here, allowing any string to be entered.

You can acchieve this by removing this block and adding `--default-addr-pool` to the option list in https://github.com/selansen/cli/blob/14dc79f6d3604f474efc500f665574aaac9d3808/contrib/completion/bash/docker#L3664.